042-SAPOnAzure/Student/07-PowerQuery.md

```diff
@@ -0,0 +1,85 @@
+# Challenge 7: Analytical Dashboard Using SAP S/4 Hana DB
+[< Previous Challenge](./06-Start-Stop-Automation.md) - **[Home](../README.md)** 
+
+# Introduction  
+
+Contoso inc. is a manufacturing company planning to modernize its IT infrastructure. As a part of that efforts, they want to Generate self-service report from SAP HANA DB tables using Microsoft power BI desktop client and develop a self-service report for visualization and analysis. 
+
+ 
+
+# Description 
+    
+  
+  This exercise includes connecting power BI desktop client to HANA DB (using direct query with DB user and password), Extract data from the required tables (using semantic SQL query) and Generate self-service Job status report.
+  
+  A high level architectural diagram can be found below.
+  
+  
+![image](https://user-images.githubusercontent.com/81314847/115074617-eefa6580-9ec7-11eb-9eaa-ca09b29708cb.png)
+ 
+
+
+## Tips
+ 
+
+   * Download and install SAP HANA client for windows ( x64 -- Coach will provide software link to download)
+    
+   * Download Power  BI  desktop from windows power app platfrom 
+    
+   * Leverage SAP HANA studio (from Challenge 4) to create the user and required roles and previleges to
+     execute the select query to populate the power BI desktop client. 
+     
+   * For the HANA DB create the SCHEMA user with admin rights on SAP SCHEMA and assign appropriate roles and        privileges to extract the data. 
+   
+   * New user ID in HANA is able to connect to Tenant database where SAP application DATA is residing              including the SAP Job log table TBTCO.
+
+     Note: Below link gives you more details on what is system DB and tenant DB in SAP HANA database.       
```

Comment:
Please uppercase the "NOTE:" and make it bold.

Reply:
done.